src/main/resources/swagger.api/schemas/tenant.yaml

```diff
@@ -47,3 +47,20 @@ TenantCollection:
   required:
     - tenants
     - totalRecords
+
+TenantDeleteRequest:
+    type: object
+    properties:
+      deleteType:
+        type: string
+        enum: [ "HARD", "SOFT" ]
+      deleteOptions:
+        type: object
+        properties:
+          deleteInternalData:
+            type: boolean
+          deleteRelatedShadowUsers:
+            type: boolean
```

Comment:
lets add 1 more parameter here: deleteUsersUserTenants.
These boolean flags already false by default, but even for more transeperency lets add this into decalration here like default: false. It should also improve documentation, that is generated from this file.